code/modules/mob/living/carbon/shock.dm

```diff
@@ -51,7 +51,7 @@
 		. += organ.internal_wound_hal_dam
 		if(organ && (organ.is_broken() || (!BP_IS_ROBOTIC(organ) && organ.open)))
 			. += 25
-		. *= max((get_specific_organ_efficiency(OP_NERVE, organ.organ_tag)/100), 0.5)
+		. *= clamp((get_specific_organ_efficiency(OP_NERVE, organ.organ_tag)/100), 0.5, 1)
```

Comment:
```suggestion
		. *= clamp((get_specific_organ_efficiency(OP_NERVE, organ.organ_tag)/100), 0.5, 1.5)
```

Reply:
Can we test 1.25? I still dislike removing the drawback of stuffing extra stronger nerves into lims

Reply:
> Can we test 1.25? I still dislike removing the drawback of stuffing extra stronger nerves into lims

That is not how the math works there I am afraid. Every multpilier you set here is modified by a 7 expononant essentially. If we set it to 1.25 then Alvua will recieve 4.7 times more pain while exalts will recieve 1.4 times more.
Without any cap, if you were to put extra 2 nerves into each leg then you would recieve 9 times more pain across the whole body.

This math loops afterall, this is why you get funni numbers.